frontend/src/app/main/ui/workspace/comments.cljs

```diff
@@ -57,20 +57,22 @@
       [:span.label (tr "labels.hide-resolved-comments")]]]))
 
 (mf/defc comments-sidebar
-  []
+  [{:keys [users threads page-id]}]
   (let [threads-map (mf/deref refs/threads-ref)
         profile     (mf/deref refs/profile)
-        users       (mf/deref refs/users)
+        users       (or users (mf/deref refs/users))
```

Comment:
From one render to other, `users` value can change from nil to something non-nil and backwards? if YES, this will break the react hooks rules.
The correct use here is `mf/deref` first, then do the OR, ensuring strict order of execution of hooks independently if `users` value.